snooty/specparser.py

```diff
@@ -262,6 +262,35 @@ def create_role(self) -> Role:
         )
 
 
+@checked
+@dataclass
+class TaxonomySpec:
+
+    TAXONOMY: ClassVar[Dict[str, Any]] = {}
```

Comment:
Can we type this using `flutter.check_type()` and `flutter.checked`? `Any` is necessary sometimes, but shouldn't be used for greenfield development.

e.g.
```
@checked
@dataclass
class GenreDefinition:
    name: str

@checked
@dataclass
class TaxonamySpec:
    name: str
    genres: List[GenreDefinition]

# etc.

taxonomy = check_type(TaxonomySpec, tomli.loads(...)
```